lib/switchtec.c

```diff
@@ -1055,10 +1055,10 @@ static int write_parsed_log(struct log_a_data log_data[],
 
 	if (entry_idx == 0) {
 		if (log_type == SWITCHTEC_LOG_PARSE_TYPE_APP)
-			fputs("   #|Timestamp                |Module       |Severity |Event\n",
+			fputs("   #|Timestamp                |Module       |Severity |Event ID|Event\n",
 		      	      log_file);
```

Comment:
Appending a white space to 'Event ID' will align better with current output style.